regression-test/suites/cast_double_to_decimal/test_cast_double_to_decimal.groovy

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+suite("test_cast_double_to_decimal") {
+    def db = "test_double"
+    sql "CREATE DATABASE IF NOT EXISTS ${db}"
+    sql "use ${db}"
```

Comment:
We don't need to use these 3 lines. Actually, regression test framework will create db automaticlly. You just need to create table.
```suggestion
     def tableName = "tbl_test_xxx"
    sql """DROP TABLE IF EXISTS ${tableName}"""
    sql """
            CREATE TABLE IF NOT EXISTS ${tableName} (xxx)
            """
           
```